app/src/atoms/Tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+import {
+  renderWithProviders,
+  TOOLTIP_TOP,
+  SPACING,
+} from '@opentrons/components'
+import { Tooltip } from '..'
+
+const render = (props: React.ComponentProps<typeof Tooltip>) => {
+  return renderWithProviders(<Tooltip {...props} />)[0]
+}
+
+const placement = TOOLTIP_TOP
+const id = 'Tooltip_123'
+const tooltipRef = jest.fn()
+const tooltipStyle = { position: 'absolute', left: SPACING.spacing2 } as const
+const arrowRef = jest.fn()
+const arrowStyle = { position: 'absolute', left: SPACING.spacing3 } as const
+
+const MockTooltipProps = {
+  id: id,
+  placement: placement,
+  arrowStyle: arrowStyle,
+  arrowRef: arrowRef,
+  visible: true,
+  ref: tooltipRef,
+  style: tooltipStyle,
+}
+
+describe('Tooltip', () => {
+  let props: React.ComponentProps<typeof Tooltip>
+
+  beforeEach(() => {
+    props = {
+      children: 'mock children' as React.ReactNode,
+      tooltipProps: MockTooltipProps,
+      key: 'mock key',
+    }
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+
+  it('renders correct children when the tooltip is visible', () => {
+    const { getByText } = render(props)
+    const tooltip = getByText('mock children')
+    expect(tooltip).toBeInTheDocument()
+    expect(tooltip).toHaveStyle('position: absolute')
+    expect(tooltip).toHaveStyle('left: 0.25rem')
+    expect(tooltip).toHaveStyle('background: #16212d')
+    expect(tooltip).toHaveStyle('color: #fff')
+    expect(tooltip).toHaveStyle('width: 8.75rem')
+    expect(tooltip).toHaveStyle('font-size: 0.625rem')
+    expect(tooltip).toHaveAttribute('role', 'tooltip')
```

Comment:
You might be able to use our constants in here so like:

```suggestion
    expect(tooltip).toHaveStyle('position: absolute')
    expect(tooltip).toHaveStyle('left: 0.25rem')
    expect(tooltip).toHaveStyle('background: darkBlackEnabled')
    expect(tooltip).toHaveStyle('color: white')
    expect(tooltip).toHaveStyle('width: 8.75rem')
    expect(tooltip).toHaveStyle('font-size: 0.625rem')
    expect(tooltip).toHaveAttribute('role', 'tooltip')
```

though I found sometimes it doesn't work for whatever reason 🤷